javascript/javascript2/week1/hyfBay/myHyf.js

```diff
@@ -0,0 +1,20 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
+
+function renderProducts(products) {
+    products.forEach(product => {
+        let myProduct = document.createElement("li");
+        let myPrice = document.createElement("li");
+        let myRating = document.createElement("li");
+        myProduct.innerHTML = "<h2>" + product.name + "</h2>";
+        document.querySelector(".my-ul").appendChild(myProduct);
+        myPrice.innerHTML = "Price: " + product.price;
+        document.querySelector(".my-ul").appendChild(myPrice);
+        myRating.innerHTML = "Rating: " + product.rating;
+        document.querySelector(".my-ul").appendChild(myRating);
```

Comment:
Every call to `document.querySelector(".my-ul")` takes time to execute. To save processing time, save the result in a variable.
```suggestion
        let myUl = document.querySelector(".my-ul");
        myUl.appendChild(myProduct);
        myPrice.innerHTML = "Price: " + product.price;
        myUl.appendChild(myPrice);
        myRating.innerHTML = "Rating: " + product.rating;
        myUl.appendChild(myRating);
```

Reply:
Thanks for the comments. I'll make the changes.